prombench/manifests/prombench/stateful-set.yaml

```diff
@@ -43,7 +43,7 @@ spec:
         imagePullPolicy: Always
         args:
         # sleep forever after make deploy succeeded
-        - "make deploy && echo 'deploy succeeded, now sleeping' && while sleep 86400; do :; done"
+        - "make deploy && touch /tmp/READY && echo 'deploy succeeded, now sleeping' && while sleep 86400; do :; done"
```

Comment:
can you also comment the workflow with `/tmp/READY` why is it created and than why it is removed etc.